pipeline/run_all.sh

```diff
@@ -40,39 +40,55 @@ python filterer.py --globals ${paramfile} --sims
 echo "Running filterer for data"
 echo "-------------------------"
 python filterer.py --globals ${paramfile} --data
+#OMP_NUM_THREADS=2 mpirun -n 4 python filterer.py --globals ${paramfile} --data
```

Comment:
My guess is that mpirun is never used at this stage, now that we will use slurm script for toast filtering,